### PR TITLE
When the 32-entry limit is met, the right-most entry is removed

### LINKED ISSUE #440: How should tracestate list members be truncated when they exceed 32?
The tracestate header field is specified to contain at most 32 list-members. What is the expected behavior when another list-member is added?

Should [the truncation schema](https://www.w3.org/TR/trace-context-1/#tracestate-limits) be used to make room for the new member or is that just the action that needs to be performed when character limits are exceeded?

OpenTelemetry is currently adding support for the tracestate and [this question came up there](https://github.com/open-telemetry/opentelemetry-go/pull/1340#discussion_r533796761) for context.

### PR DESCRIPTION
**Pull request by dyladan**
Fixes #440 

/cc @MrAlias

***
<a href="https://pr-preview.s3.amazonaws.com/dynatrace-oss-contrib/trace-context/pull/448.html" title="Last updated on Jan 12, 2021, 9:12 PM UTC (05a18c3)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/trace-context/448/56c7111...dynatrace-oss-contrib:05a18c3.html" title="Last updated on Jan 12, 2021, 9:12 PM UTC (05a18c3)">Diff</a>

### CODE REVIEW

File: spec/20-http_request_header_format.md
Comment (yurishkuro) [inline spec/20-http_request_header_format.md:313]: perhaps worth defining that we mean by "entry length" (we have somewhat odd rules for handling whitespace, namely that leading whitespace in the value must be preserved)

**Review by yurishkuro — APPROVED** [review]

**Review by SergeyKanzhelev — APPROVED** [review]

**Review by MrAlias — APPROVED** [review]

**Comment by dyladan** [comment]
I can't find anywhere what the requirements are to merge a PR. I would merge this but I'm not sure if I'm allowed? There should be some merge procedure documented.

**Comment by SergeyKanzhelev** [comment]
> I can't find anywhere what the requirements are to merge a PR. I would merge this but I'm not sure if I'm allowed? There should be some merge procedure documented.

I think the procedure is to get enough "diverse" approvals and make sure PR was active for at least couple days. So next time feel free to merge.

**Pull request merged**
